### PR TITLE
chore/Remove option to change production branch

### PR DESCRIPTION
**Pull request by joshenlim**
- Fix loading state for github project picker
- Fix vercel/github icons not showing on project cards if integrated
- Remove option to change production branch

### CODE REVIEW

**Comment by MildTomato** [comment]
was about to comment did you forget to push @joshenlim 😂

File: studio/components/interfaces/Settings/Integrations/GithubIntegration/GitHubIntegrationConnectionForm.tsx [hunk 1]
```diff
@@ -134,11 +137,21 @@ const GitHubIntegrationConnectionForm = ({
         <p className="text-xs text-light mb-3">
           All other branches will be treated as Preview branches
         </p>
+
+        <Alert_Shadcn_ className="mb-4 w-96">
+          <AlertTitle_Shadcn_ className="text-sm">
+            We currently don’t allow changing production branch
```
Comment (MildTomato) [inline studio/components/interfaces/Settings/Integrations/GithubIntegration/GitHubIntegrationConnectionForm.tsx:143]:
```suggestion
            Changing Production branch coming soon
```

**Review by MildTomato — COMMENTED** [review]
Maybe we also need to add a Alert for the disabled directory input as well ?

File: studio/components/interfaces/Settings/Integrations/GithubIntegration/GitHubIntegrationConnectionForm.tsx (hunk 1 above)
Comment (joshenlim) [reply studio/components/interfaces/Settings/Integrations/GithubIntegration/GitHubIntegrationConnectionForm.tsx:143]: opting for `Changing Git branch for production branch coming soon`?
"branching production branch" sounds a little odd to me haha

File: studio/components/interfaces/Settings/Integrations/GithubIntegration/GitHubIntegrationConnectionForm.tsx [hunk 2]
```diff
@@ -134,11 +137,21 @@ const GitHubIntegrationConnectionForm = ({
...
+          </AlertTitle_Shadcn_>
+          <AlertDescription_Shadcn_ className="text-xs">
+            You will need to disable branching and opt back in to choose a new production branch
```
[the remark this answers is not in the document]
Comment (joshenlim) [reply studio/components/interfaces/Settings/Integrations/GithubIntegration/GitHubIntegrationConnectionForm.tsx:146]: are we standardizing to capitalize "Production Branch" and "Branching"?

**Comment by joshenlim** [comment]
> Maybe we also need to add a Alert for the disabled directory input as well ?

Any messaging you have in mind?

File: studio/components/interfaces/Settings/Integrations/GithubIntegration/GitHubIntegrationConnectionForm.tsx [hunk 3]
```diff
@@ -209,6 +223,17 @@ const GitHubIntegrationConnectionForm = ({
           <FormDescription_Shadcn_ className="text-xs text-lighter mb-3">
             Migration and seed SQL files will be run from this directory.
           </FormDescription_Shadcn_>
+
+          <Alert_Shadcn_ className="mb-4 w-96">
+            <AlertTitle_Shadcn_ className="text-sm">
+              Changing Supabase directory is currently not supported
+            </AlertTitle_Shadcn_>
+            <AlertDescription_Shadcn_ className="text-xs">
+              Your migration and seed SQL files will need to be within the /supabase directory of
```
Comment (MildTomato) [inline studio/components/interfaces/Settings/Integrations/GithubIntegration/GitHubIntegrationConnectionForm.tsx:232]:
```suggestion
              You will need to disable Branching and opt back into Branching to change the Production Branch. 
```

**Review by MildTomato — APPROVED** [review]

**Pull request merged**